singleFrameProcCoreTasks/calibimg.rst

```diff
@@ -0,0 +1,216 @@
+#############
+CalibrateTask
+#############
+
+Given a properly characterized exposure (which means one with a PSF
+determined and shipped along with the image, commonly done previous to
+running this Task by :doc:`CharacterizeImage <charimg>`), detect
```

Comment:
Okay used this wording (though I will say that in this case I think I personally didn't find that sentence to be overly verbose, myself..)